ui/package.json

```
@@ -87,6 +87,7 @@
         "yarn-deduplicate": "^6.0.2"
     },
     "resolutions": {
-        "semver": "^7.5.2"
+        "semver": "^7.5.2",
+        "axios": "1.6.0"
```

this doesn't need a `resolution`, as I mentioned in a [previous PR](https://github.com/argoproj/argo-workflows/pull/11641#issuecomment-1693869029). you can change the lockfile directly to update it.

A `resolution` is an override, but we don't need to override this as it's a SemVer-compatible update

```suggestion
        "semver": "^7.5.2"
```